MyArrayList.java

```diff
@@ -1 +1,100 @@
+package com.company;
 
+public class MyArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private int position = 0;
+    private Object[] array = new Object[DEFAULT_CAPACITY];
+    private StringBuilder stringBuilder = new StringBuilder();
+    private int size = array.length;
+
+    public void printArrayList() {
+        System.out.println("My array list: ");
+        stringBuilder.setLength(0);
+        for (int j = 0; j < size; j++) {
+            stringBuilder.append(array[j]).append(", ");
+        }
+        System.out.println(stringBuilder);
+    }
+
+    @Override
+    public void add(T value) {
+        check(position);
+        add(value, position);
+        position++;
+    }
+
+    private void check(int i) {
+        if (i >= size) {
+            size += DEFAULT_CAPACITY / 2;
```

Comment:
size += DEFAULT_CAPACITY / 2;  ->
size += size / 2;

Reply:
Ok

Reply:
But i think we need to increase out array  by 1.5 times. Because size can be more or less 10. 
Maybe i'm not right.

Reply:
we need increase array in 1.5 times from the current length